src/linagora.esn.unifiedinbox/i18n/en.json

```diff
@@ -203,6 +207,7 @@
   "Test": "Test",
   "Thanks a lot for your message. I am currently out of office and will get back to you soon.": "Thanks a lot for your message. I am currently out of office and will get back to you soon.",
   "then": "then",
+  "There is no current experiments. Come back later !": "There is no current experiments. Come back later !",
```

Comment:
```suggestion
  "There are no current experiments. Come back later!": "There are no current experiments. Come back later!",
```

Reply:
@rezk2ll you're right this english is not good. What do you think of `No experiment is currently available. Come back later!` ?

Reply:
sounds good.